openreview/conference/builder.py

```diff
@@ -300,8 +300,8 @@ def get_bid_id(self, group_id):
     def get_recommendation_id(self, number = None):
         return self.get_invitation_id(self.recommendation_name, number)
 
-    def get_registration_id(self):
-        return self.get_invitation_id(self.registration_name)
+    def get_registration_id(self, committee_id = None):
```

Comment:
committee_id should be a mandatory parameter, you are using always as part of the invitation id